apps/web/src/views/Home/components/Banners/MultipleBanner.tsx

```diff
@@ -115,7 +112,7 @@ const MultipleBanner: React.FC<React.PropsWithChildren> = () => {
   }, [bannerList, swiperRef])
 
   return (
-    <BannerPlaceHolder walletConnected={Boolean(account) && chainId === ChainId.BSC}>
+    <BannerPlaceHolder walletConnected={false}>
```

Comment:
If `walletConnected ` always false. Can u clear the walletConnected  code in css?